### PR TITLE
Elivepatch

### PR DESCRIPTION
**Pull request by aliceinwire**
initial version for elivepatch-server and elivepatch-client

### CODE REVIEW

File: sys-devel/elivepatch-client/elivepatch-client-0.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Live patch installer client working with elivepatch-server"
+HOMEPAGE="https://wiki.gentoo.org/wiki/User:Aliceinwire/elivepatch"
```
Comment (gktrk) [inline sys-devel/elivepatch-client/elivepatch-client-0.0.1.ebuild]: Update the HOMEPAGE to point to https://wiki.gentoo.org/wiki/Elivepatch

File: sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Live patch building server with RESTFul Api for elivepatch-client"
+HOMEPAGE="https://wiki.gentoo.org/wiki/User:Aliceinwire/elivepatch"
```
Comment (gktrk) [inline sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild]: https://wiki.gentoo.org/wiki/Elivepatch

File: sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+SRC_URI="https://github.com/aliceinwire/elivepatch-server/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
```
[the remark this answers is not in the document]
Comment (aliceinwire) [reply sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild]: werkzeug is taken with flask dependency

**Comment by aliceinwire** [comment]
I think the resources install location need to be fixed by setup.py

**Comment by aliceinwire** [comment]
after latest changes to elivepatch-server
https://gist.github.com/aliceinwire/9c5163bfdc62dd995505bbdeb0b01b82

**Comment by gktrk** [comment]
Why is the category `sys-devel`? I don't think this belongs here. It's not used for a system-level development. `sys-apps` sounds like a better category to me.

File: sys-devel/elivepatch-client/elivepatch-client-0.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+dev-python/git-python[${PYTHON_USEDEP}]
+dev-python/requests[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (gktrk) [inline sys-devel/elivepatch-client/elivepatch-client-0.0.1.ebuild]: These entries should have a single level indent after DEPEND/RDEPEND

File: sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+dev-python/werkzeug[${PYTHON_USEDEP}]
+dev-python/flask[${PYTHON_USEDEP}]
+dev-python/flask-restful[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (gktrk) [inline sys-devel/elivepatch-server/elivepatch-server-0.0.1.ebuild]: These entries should have a single level indent after DEPEND/RDEPEND

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/4c4cea7e919...e6c60412216. Congratulations!

**Pull request closed without merging**